app/models/genre.rb

```diff
@@ -0,0 +1,12 @@
+class Genre < ApplicationRecord
+  has_and_belongs_to_many :books
+
+  validates :name, presence: true
+
+  scope :ordered, -> { order(name: :asc) }
```

Comment:
Could you update this ordering so it is case insensitive? Like in Book in your other PR